tests/test_ops.py

```diff
@@ -1395,14 +1412,29 @@ def test_mathtransform():
         # {"opName": "zeros", "outName": "emptyArrayTests/zeros/ones_rank3", "varShapes":[[3]], "varTypes":["int32"], "varInit":["fixed_0_0_3"], "dtype":tf.int64},
         # {"opName": "zeros_like", "outName": "emptyArrayTests/zeros_like/rank1", "varShapes":[[0]], "varTypes":["float32"], "varInit":["empty"]},
         # {"opName": "zeros_like", "outName": "emptyArrayTests/zeros_like/rank2", "varShapes":[[2,0]], "varTypes":["float64"], "varInit":["empty"]},
+        # {"opName": "zeros_like", "outName": "zeros_like/rank2_float16", "varShapes": [[3, 12]],"varTypes": ["float16"], "varInit": ["uniform"]},
+        #{"opName": "zeros_like", "outName": "zeros_like/rank2_float32", "varShapes": [[3, 12]], "varTypes": ["float32"], "varInit": ["uniform"]},
+        #{"opName": "zeros_like", "outName": "zeros_like/rank2_float64", "varShapes": [[3, 12]], "varTypes": ["float64"], "varInit": ["uniform"]},
+        #{"opName": "zeros_like", "outName": "zeros_like/rank2_int32", "varShapes": [[3, 10]], "varTypes": ["int32"], "varInit": ["uniform_int10"]},
+        #{"opName": "zeros_like", "outName": "zeros_like/rank2_int64", "varShapes": [[3, 10]], "varTypes": ["int64"], "varInit": ["uniform_int10"]},
```

Comment:
What about adding an empty array case? Rank 0 scalar array case? Boolean case?
It also has optional dtype arg, for output type, we'd want at least one test setting that (to different to input type)

Reply:
Empty arrays tests were present before. dtype tests added.